src/main/java/com/google/cloud/tools/appengine/api/devserver/DefaultRunConfiguration.java

```diff
@@ -48,6 +49,7 @@
   private String defaultGcsBucketName;
   private Boolean clearDatastore;
   private File datastorePath;
+  private Map<String, String> environment;
```

Comment:
I'm not sure how maven handles maps as inputs. I need to check it out, but how does the command line handle this for dev_appserver2? which actually has an --env_var parameter.